### PR TITLE
[BUG]Fix restart issue of csi plugin in k8s 1.14

### LINKED ISSUE #220: [BUG]fuse.csi.fluid.io not found in the list registered CSI drivers when node restart in k8s 1.15.1
**What is your environment(Kubernetes version, Fluid version, etc.)**
kubernetes:v1.15
fluid:0.3.0

**Describe the bug**
I deploy runtime and dataset in node1,everything goes well.when i restart node1,runtime and dataset is running,csi-nodeplugin-fluid-xxx is also running,when i deploy a pod in this node,the pod log shows MountVolume.MountDevice failed for volume "zj-lmdb" : driver name fuse.csi.fluid.io not found in the list registered CSI drivers 

**What you expect to happen:**

**How to reproduce it**
restart the runtime node
**Additional Information**

### PR DESCRIPTION
**Pull request by cheyang**
### Ⅰ. Describe what this PR does

Fixes

1.  driver name fuse.csi.fluid.io not found in the list of registered CSI drivers
2.  move `alluxio-mnt` out of `/var/lib/kubelet` which will cause mount leak, you can check with 
```
cat /proc/mounts |grep alluxio-mnt
```

3.  Make the ALLUXIO_ROOT configurable
4. Fix the check of IsMount 

### Ⅱ. Does this pull request fix one issue?

fixes #220 

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Review by xieydd — COMMENTED** [review]
Except a slip of the pen.
/lgtm

File: csi/shell/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -xe
+
+rm -f /var/lib/kubelet/csi-plugins/fuse.csi.fluid.io、csi.sock
```
[the remark this answers is not in the document]
Comment (cheyang) [reply csi/shell/entrypoint.sh:4]: Thanks very much for spotting it!

File: pkg/csi/fuse/utils.go [hunk 2]
```diff
@@ -41,3 +44,27 @@ func checkMountReady(fluidPath string) error {
...
 }
+
+func isMounted(absPath string) (bool, error) {
+	glog.Infof("abspath: %s\n", absPath)
+	_, err := os.Stat(absPath)
+	if err != nil {
+		return false, err
+	}
+
+	file, err := ioutil.ReadFile("/proc/mounts")
```
Comment (TrafalgarZZZ) [inline pkg/csi/fuse/utils.go:55]: Can this be replaced with mount.List()?

Comment (cheyang) [reply pkg/csi/fuse/utils.go:55]: I think so. I used the code from https://github.com/containers/build/blob/v0.4.0/util/mounts.go#L25 .

Comment (cheyang) [reply pkg/csi/fuse/utils.go:55]: Maybe we can enhance it in the next PR, because I've tested the current code for several cases.

**Pull request merged**